### PR TITLE
Layout fixes and css cleanup

### PR DESCRIPTION
**Pull request by aabidsofi19**

### CODE REVIEW

**Review by jonemo — COMMENTED** [review]
Starting off the review with a few inline comments and questions.

I also noticed a few visual changes where I am not sure if they were intentional or by accident:

* Alignment of text and image on home page has changed and is quite strange on some screen sizes
* Font family and style of "Jonas Neubert" above navigation has changed
* The shade of red for links and text-highlight is different now, looks kind of orange-red. See related inline comment in `css/main.css`
* Letter spacing and line spacing has changed. The details of this are not super important to me, but overall the text density has gone down too much with this change.

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,1947 @@
+/*
```
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply assets/css/style.css:1]: yeah sure . it will be auto generated on the go

[the remark this answers is not in the document]
Comment (aabidsofi19) [reply assets/css/style.css:1]: done

File: assets/css/main.css [hunk 2]
```diff
@@ -3,32 +3,29 @@
...
+        @apply tracking-wider font-light  ;  /* bring back spacing from pre-Google fonts version of Quicksand */
+    } 
+    
+    main a, main a:visited, main a:link {
+        @apply text-red-600 decoration-transparent font-medium;
+        
+    }
+
+    main ::selection {
+
```
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply assets/css/main.css:18]: I guess there was  some problem with format on save and I  forgot to run prettier before commit .

File: layouts/index.html [hunk 3]
```diff
@@ -1,81 +1,102 @@
...
+        width="376"
+        alt=""
+      />
     </div>
 
-    <div id="aboutjonas">
-      <div>
-        <p>Hi, I'm Jonas.</p>
+    <div class="w-full">
+      <div class="w-full md:w-1/2">
```
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply layouts/index.html:36]: yeah . previously we were using 42vw for large screens . Thats why i am using a full width container and then inside a half width  . But the breakpoindt should have been around xl ..

[the remark this answers is not in the document]
Comment (aabidsofi19) [reply layouts/index.html:36]: done

File: layouts/index.html [hunk 4]
```diff
@@ -1,81 +1,102 @@
...
-        id="speakingbubble1"
+        class="absolute left-72"
+        style="bottom: 278px;"
         src="images/speakingbubble.png"
         width="20"
         height="10"
       />
-      <img id="couch" src="images/couch.png" height="264" width="376" alt="" />
+      <img
+        class="absolute left-0 bottom-0  max-w-full"
```
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply layouts/index.html:27]: prettier issue  : (

[the remark this answers is not in the document]
Comment (aabidsofi19) [reply layouts/index.html:27]: done

File: layouts/partials/tags.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<ul class="flex items-center gap-4">
+       {{ range .Params.tags }}
```
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply layouts/partials/tags.html:2]: done

File: postcss.config.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply postcss.config.js:1]: i am using postcss  to  use [autoprefixer](https://github.com/postcss/autoprefixer) to generate vendor prefixes .    

But forgot to update the build script . 

for local development  we dont  dont need to run postcss  .

[the remark this answers is not in the document]
Comment (aabidsofi19) [reply postcss.config.js:1]: done

File: assets/css/main.css [hunk 7]
```diff
@@ -3,32 +3,29 @@
...
-   yellow = #ffe400
-*/
+@layer base {
+    
+    html{
+        @apply tracking-wider font-light  ;  /* bring back spacing from pre-Google fonts version of Quicksand */
+    } 
+    
+    main a, main a:visited, main a:link {
+        @apply text-red-600 decoration-transparent font-medium;
```
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply assets/css/main.css:13]: fixed : i should have been colors .

File: assets/css/main.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply assets/css/main.css:18]: fixed

File: content/talks/Extreme-Bikesharing.md [hunk 8]
```diff
@@ -11,6 +11,8 @@ Extreme Bikesharing: How I used Python to plan an epic Ford GoBike trip
 
 Presented at [SF Python Talk Night on Jan 8, 2020](https://www.meetup.com/sfpython/events/xkwxvqybccblb/) and at [Full Stack Developer Meetup on Jan 14, 2020](https://www.meetup.com/Full-Stack-Development-Meetup-Group/events/267022346/).
 
+
+
```
[the remark this answers is not in the document]
Comment (aabidsofi19) [reply content/talks/Extreme-Bikesharing.md:15]: >I was  accidently testing  inline code fixes  here :) .

[the remark this answers is not in the document]
Comment (aabidsofi19) [reply content/talks/Extreme-Bikesharing.md:15]: done

**Comment by aabidsofi19** [comment]
For the text density issue . you were previously using  0.05 em as letter spacing  but tailwind provides wide 0.025 and wider 0.05 ( which we are using now but may be too much ) .  we can use the hardcoded value if you like the old spacing or  use wide ,

**Pull request merged**